Makefile.am

```diff
@@ -82,13 +82,13 @@ TEST_INST_BUILD_DIR = $(BUILD_RESULT_DIR)/02-test-install
 # build/run tweaks for all containers, such as --cap-add
 CONTAINER_ENGINE ?= podman
 CONTAINER_BUILD_ARGS ?=
-CONTAINER_TEST_ARGS ?=
+CONTAINER_TEST_ARGS ?= --rm -v $(srcdir):/anaconda:Z
```

Comment:
Admittedly I liked [this version](https://github.com/martinpitt/anaconda/commit/d68ba304861d) better as it simplifies the user-facing invocation for a shell, but that's not a veto.

I intended the generic `CONTAINER_*_ARGS` to be empty by default and user-definable, so that the user can add stuff like `--cap-add`, `--device`, or additional `-v`s without breaking the default behaviour.